docs/reference/contributing/target/flash.md

```diff
@@ -1,8 +1,10 @@
-### Flash
+### Flash and Bootloader
 
-Update target to support bootloader.
+Update target to support bootloader:
 
 1. Update linker script.
+1. Add required metadata to targets.json
```

Comment:
Please put "targets.json" in code format.

Reply:
And a `.`, right?

Reply:
Yes, thanks :)